flink-sql-runner/src/test/java/com/github/streamshub/flink/SqlRunnerTest.java

```diff
@@ -0,0 +1,42 @@
+package com.github.streamshub.flink;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SqlRunnerTest {
+
+    @Test
+    void testParseStatements() {
+        String statements = "CREATE TABLE Table1 ( field STRING, field1 STRING ) WITH ( 'connector' = 'my-source-connector', 'path' = '/test1/data' ); CREATE TABLE Table2 ( field STRING, field2 STRING ) WITH ( 'connector' = 'my-sink-connector', 'path' = '/path/data2' ); INSERT INTO Table2 SELECT * FROM Table1;";
```

Comment:
Just out of curiosity - isn't there some java library that offers builder for SQL statements?

Reply:
I had a play with jOOQ (which offers a [DSL for building generic SQL strings](https://www.jooq.org/doc/3.19/manual/sql-building/)) and I don't think it has any concept of the WITH properties style Flink uses to define the additional props for a table. Also you have to tinker to make sure it formats the output the way we like, and it's resistant to emitting "string" as the data type in the different SQL dialects I tried.

I think in this case we're better off with String manipulation since the runner cares a lot about splitting on delimiters and less about the SQL validity, we want to know exactly what strings are running through it. Though I think it's good to test with some realistic looking inputs.

Reply:
Thanks @robobario for checking how jOOQ  works. In that case, I agree on using String manipulation rather a library for building SQL statements. I can update the SQL statements to look more realistic.

Reply:
That makes sense to me, thanks for explanation